src/createXcmTypes/util/createSystemToParaMultiAssets.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+
+import type { ApiPromise } from '@polkadot/api';
+
+import { findRelayChain, parseRegistry } from '../../registry';
+import { MultiAsset, MultiAssetInterior } from '../../types';
+import { isAscendingOrder } from './checkIsAscendingOrder';
+import { dedupeMultiAssets } from './dedupeMultiAssets';
+import { fetchPalletInstanceId } from './fetchPalletInstanceId';
+import { getSystemChainTokenSymbolGeneralIndex } from './getTokenSymbolGeneralIndex';
+import { sortMultiAssetsAscending } from './sortMultiAssetsAscending';
+
+/**
+ * Creates and returns a MultiAsset array for system parachains based on provided specName, assets and amounts 
+ *
+ * @param api ApiPromise[]
+ * @param amounts string[]
+ * @param specName string
+ * @param assets string[]
+ */
+export const createSystemToParaMultiAssets = (
```

Comment:
We can add this directly inside of the SystemToPara file as a seperate helper function.